ImageManager/ImageManager.py

```diff
@@ -0,0 +1,89 @@
+"""
+    Handles CRUD operations for images, and their configuration on the
+    FIWARE backend
+"""
+
+import json
+import logging
+from time import time
+from flask import request
+from flask import make_response
+from flask import Blueprint
+from utils import *
+
+from DatabaseModels import *
+from SerializationModels import *
+from TenancyManager import init_tenant_context
+
+from app import app
+
+image = Blueprint('image', __name__)
+
+LOGGER = logging.getLogger('image-manager.' + __name__)
+LOGGER.addHandler(logging.StreamHandler())
+LOGGER.setLevel(logging.INFO)
+
+global_imageid = 20
+
+
+@image.route('/image/<imageid>', methods=['GET'])
+def get_image(imageid):
+    try:
+        print(request)
+        init_tenant_context(request, db)
+        orm_image = assert_image_exists(imageid)
+        result = {
+            "label": orm_image.label,
+            "fw_version": orm_image.fw_version,
+            "hw_version": orm_image.hw_version,
+            "sha1": orm_image.sha1
+        }
+        return make_response(json.dumps(result), 200)
+    except HTTPRequestError as e:
+        if isinstance(e.message, dict):
+            return make_response(json.dumps(e.message), e.error_code)
+        else:
+            return format_response(e.error_code, e.message)
+
+
+@image.route('/image/', methods=['POST'])
+def create_image():
+    """ Creates and configures the given image (in json) """
+    try:
+        tenant = init_tenant_context(request, db)
+        image_data, json_payload = parse_payload(request, image_schema)
+        # imageid = generate_image_id()
+        global global_imageid
+        imageid = str(global_imageid)
+        global_imageid = global_imageid + 1
+        image_data['id'] = imageid
+
+        orm_image = Image(**image_data)
+        db.session.add(orm_image)
+
+        # # TODO remove this in favor of kafka as data broker....
```

Comment:
I think those can be safely removed from code - this service should require no events of that nature being sent at this time.